src/wazuh_modules/syscollector/tests/sysCollectorFlatbuffers/sysCollectorFlatbuffers_test.cpp

```diff
@@ -0,0 +1,424 @@
+/*
+ * Wazuh SyscollectorImp
+ * Copyright (C) 2015, Wazuh Inc.
+ * August 1, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "sysCollectorFlatbuffers_test.h"
+#include "flatbuffers/util.h"
+#include "flatbuffers/idl.h"
+
+const std::string netIfaceSchemaStr
```

Comment:
We could find a way to directly read the `.fbs` files to avoid duplicating the schemas.

Reply:
Done